app/components/EmojiPicker.tsx

```diff
@@ -0,0 +1,229 @@
+import data from "@emoji-mart/data";
+import Picker from "@emoji-mart/react";
+import { SmileyIcon } from "outline-icons";
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { usePopoverState, PopoverDisclosure } from "reakit/Popover";
+import styled, { useTheme } from "styled-components";
+import { depths, s } from "@shared/styles";
+import { hexToRgb } from "@shared/utils/color";
+import Button from "~/components/Button";
+import Popover from "~/components/Popover";
+import usePickerTheme from "~/hooks/usePickerTheme";
+import useUserLocale from "~/hooks/useUserLocale";
+import { hover } from "~/styles";
+import Emoji from "./Emoji";
+import NudeButton from "./NudeButton";
+
+type Props = {
+  value?: string | null;
+  onChange: (emoji: string | null) => Promise<void>;
+  onClickOutside: () => void;
+};
+
+const locales = {
```

Comment:
No need for this mapping? It'll always be the first part of the code as this is a standard, can split by `_`